error-handling/src/main/java/com/palantir/remoting2/errors/SerializableStackTraceElement.java

```diff
@@ -30,13 +30,23 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class SerializableStackTraceElement implements Serializable {
 
-    public abstract Optional<String> getClassName();
+    public abstract String getClassName();
```

Comment:
this isn't necessarily coming from java, so we need to keep the optionality in the wire format